apps/api/v1/pages/api/credential-sync/_post.ts

```diff
@@ -6,7 +6,9 @@ import { appStoreMetadata } from "@calcom/app-store/appStoreMetaData";
 import { symmetricDecrypt } from "@calcom/lib/crypto";
 import { HttpError } from "@calcom/lib/http-error";
 import { defaultResponder } from "@calcom/lib/server";
+import { CredentialRepository } from "@calcom/lib/server/repository/credential";
```

Comment:
Good refactor! Switching to "CredentialRepository" improves modularity and organizes dependencies more effectively with the added import.